dashboard/app/models/authentication_option.rb

```diff
@@ -54,7 +54,6 @@ class AuthenticationOption < ApplicationRecord
   SILENT_TAKEOVER_CREDENTIAL_TYPES = [
     FACEBOOK,
     GOOGLE,
-    WINDOWS_LIVE,
```

Comment:
Necessary to keep accounts from switching back and forth between the two types?  If it's still possible to sign in with Windows Live, should we in theory still support silent takeover so we don't accidentally help folks create duplicate accounts?

Reply:
i removed this so users aren't switching between windowslive and microsoft_v2_auth, but since we are totally removing the ability to sign in via windowslive, that may not be necessary. i think your point about supporting silent takeover in the meantime makes sense though. i'll re-add this line for now and remove later once we aren't at risk of users still somehow signing in via windowslive